import-automation/cloudbuild/cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+- name: 'gcr.io/cloud-builders/gcloud'
```

Comment:
Perhaps add a comment explaining what this chunk of code does.  Your pull request description is probably a superset of what to write here.

Reply:
added